ReleaseNotes.md

```diff
@@ -1,5 +1,55 @@
 # Release Notes
 
+## New in Release 2.5.0
+Compression-aware training:
+- New Features:
+  - Added Patcher class from OTX, which is used for patching the models that can not be wrapped properly.
```

Comment:
Added `nncf.common.utils.patcher.Patcher` - this class can be used to patch methods on live PyTorch model objects with wrappers such as `nncf.torch.dynamic_graph.context.no_nncf_trace` when doing so in the model code is not possible (e.g. if the model comes from an external library package).

Reply:
Is it only for PyTorch, bu the way?

Reply:
> Is it only for PyTorch, bu the way?

Not only for PyTorch, but it only currently has uses for PyTorch.